echo/server/dembrane/api/stateless.py

```
@@ -267,3 +269,23 @@ async def delete_conversation(payload: DeleteConversationRequest,
+    logger.debug(f"Transcribe webhook received: {payload}")
+    try:
+        directus.update_item(
+            collection_name="conversation_chunk",
+            item_id=payload["output"]["conversation_chunk_id"],
+            item_data={
+                "transcript": payload["output"]["joined_text"],
+                "processing_status": ProcessingStatus.COMPLETED.value,
+                "runpod_job_status_link": None
+            },
+        )
+        logger.info(f"updated chunk with transcript: {payload['output']['conversation_chunk_id']} - length: {len(payload['output']['joined_text'])}")
```

Pylint (3.3.7)

[convention] Line too long (149/100)

(C0301)